src/bin/clapper-app/ui/clapper-app-uri-dialog.ui

```diff
@@ -1,21 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface domain="clapper-app">
-  <object class="AdwMessageDialog" id="dialog">
-    <property name="modal">true</property>
-    <property name="width-request">420</property>
+  <object class="AdwAlertDialog" id="dialog">
     <property name="heading" translatable="yes">Add URI</property>
     <property name="body" translatable="yes">Insert an URI to be added to playback queue</property>
     <property name="close-response">cancel</property>
     <property name="default-response">add</property>
+    <property name="follows-content-size">false</property>
```

Comment:
```suggestion
    <property name="follows-content-size">false</property>
    <property name="content-width">420</property>
```

Reply:
The URI entry is a little too narrow to input/preview link conveniently. Can we restore "width-request" to 420 as it was? AFAIU, in libadwaita the prop to set is called "content-width".

Reply:
Sounds good to me